packages/web-components/src/components/ic-dialog/ic-dialog.css

```diff
@@ -148,7 +148,7 @@
 }
 
 @media (min-width: 800px) {
-  :host([size="large"]) .content-area {
+  :host([size="large"]) .dialog:not(.disable-width-constraint) > .content-area {
```

Comment:
When i've tried this locally i've found that dialog widths were still restricted by the 'max-width' rule in the .large class (line 78).
![Screenshot 2024-02-19 at 10 09 00](https://github.com/mi6/ic-ui-kit/assets/111292477/80dc20ba-69ba-4246-bf2a-0af7643b9be2)